tests/util/test_createPREMISEventXML.py

```diff
@@ -44,7 +44,7 @@ def test_validate_eventxml(premis_args, premis_schema):
     Check produced PREMIS Event xml against schema
     """
     premis = util.createPREMISEventXML(**premis_args)
-    premis_schema.assert_(premis)
+    premis_schema.validate(premis)
```

Comment:
This will return `True` or `False`, but since we aren't making an assertion about it, whether it returns one value or the other, won't fail the test.
```suggestion
    premis_schema.assertValid(premis)
```